base/build.gradle

```diff
@@ -36,6 +36,8 @@ dependencies {
     testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.9.0' // for junit framework
 
     testFixturesImplementation("com.guardsquare:proguard-assembler:1.0.0")
+
+    implementation 'com.google.code.gson:gson:2.10.1'
```

Comment:
Can you avoid adding any dependencies? Since all users of PGC will now have this dependency even if they are not using the machine output. Can you generate the JSON yourself? Or is there some simple (small) JSON generator that can be copy & pasted (if license permits).